luigi/scheduler.py

```diff
@@ -353,10 +353,14 @@ def __init__(self, worker_id, last_active=None):
         self.info = {}
         self.disabled = False
         self.rpc_messages = []
+        self.custom_disconnect_delay = None
```

Comment:
Related to you question of if `custom_disconnect_delay` should live as an attribute of Worker... It does seem a little odd here since all the other attributes of this class are related to tracking work.

Given that the value this would be overriding, `config.worker_disconnect_delay`, is a scheduler config value, that perhaps this configuration should be a scheduler rpc update. Thoughts? (it is already a bit confusing the overlap between scheduler and worker config).

Reply:
You lost me at 
> this configuration should be a scheduler rpc update.

However, i found a reasonably elegant way to save the frontend from dealing with the possible Null value. `worker_list` can obtain the current value. 

I don't see a better way right now. I'm not sure if there's a good way to make Scheduler construct Workers with a configuration value. Also, it seems that it would be complicated to make Worker weakref its Scheduler (to obtain scheduler config in, say, a property getter). 

https://github.com/koo5/luigi/commit/4c220d67728bd9ed91917d08c327dec37cc0584e

Also, nevermind about using `Worker.info`, as that seems to be for passing information from workers to scheduler.